setupext.py

```diff
@@ -17,7 +17,8 @@
 import versioneer
 
 
-PY3 = (sys.version_info[0] >= 3)
+PY3 = (sys.version_info.major >= 3)
```

Comment:
The named components do not exist in 2.6, if the 1.5.x line still feels like supporting that.